tensorflow/python/keras/utils/vis_utils_test.py

```diff
@@ -103,6 +104,129 @@ def test_plot_model_with_add_loss(self):
     except ImportError:
       pass
 
+  def test_dot_layer_range(self):
+    model = efficientnet.EfficientNetB0()
+
+    # Case 1: when layer names in normal (increasing) order
```

Comment:
The case 1 and 2 body are actually the same (only the input has the different order). Please convert this to a parameterized test to avoid all the duplications.

Reply:
done